src/pathfinding_service/api.py

```diff
@@ -176,6 +176,7 @@ def post(self, token_network_address: str) -> Tuple[dict, int]:
             value=path_req.value,
             reachability_state=self.pathfinding_service.matrix_listener.user_manager,
             max_paths=path_req.max_paths,
+            user_manager=self.pathfinding_service.matrix_listener.user_manager,
```

Comment:
If I see correctly, you replaced the reachability state with the user_manager. Why do we need to keep both?